database.config.example.js

```diff
@@ -0,0 +1,10 @@
+export default `{
+  // The host where the mariaDB is located
+  "host": "127.0.0.1",
```

Comment:
Hmm, yeah might be a legacy thing because I was changing methods... Yeah no it doesn't really make much sense I guess.

It's because I was using the reading writing files etc... Yeah I'll definitely change that